src/sig_stfl/lms/external/hss_alloc.c

```diff
@@ -417,7 +418,8 @@ signed long initial_mem_target = mem_target; /* DEBUG HACK */
         return 0;
     }
 #if 0
-printf( "Allocation = %ld\n", initial_mem_target - mem_target + best_mem ); /* DEBUG HACK */
+printf( "Allocation = %ld
```

Comment:
Did something go wrong with the `\n` symbol here?

Reply:
Yep, will fix.